grammar/argextend.mlp

```diff
@@ -11,8 +11,14 @@ open Q_util
 let loc = Ploc.dummy
 let default_loc = <:expr< Loc.ghost >>
 
+IFDEF STRICT THEN
+  let ploc_vala x = Ploc.VaVal x
+ELSE
+  let ploc_vala x = x
+END
+
```

Comment:
Grrr, OK, fine with me. FTR, the `q` prefix had something to do with quotations I believe.